ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts

```diff
@@ -130,3 +134,28 @@ function findMax(timeSeries: EChartsTimeSeries[]) {
   });
   return max;
 }
+
+/**
+ * Converts Perses panel y_axis from dashboard spec to ECharts supported yAxis options
+ */
+export function convertPanelYAxis(inputAxis: YAxisOptions = {}): YAXisComponentOption {
+  const yAxis: YAXisComponentOption = {
+    show: inputAxis?.show ?? DEFAULT_Y_AXIS.show,
+    min: inputAxis?.min,
+    max: inputAxis?.max,
+  };
+
+  if (inputAxis?.min === undefined) {
+    // Sets minimum axis label relative to data instead of zero.
+    yAxis.min = (value) => {
+      // https://echarts.apache.org/en/option.html#yAxis.min
+      if (value.min <= 1) {
+        return 0;
+      }
```

Comment:
Oh just had a thought that this could be weird if the chart goes negative!! 🐛 ?

Reply:
(In other words, if the minimum is -1000 or something, we wouldn't want to chart to cut off at zero...)

Reply:
Oh wow, yeah you're right, thanks for catching this! It's fixed now:

<img width="832" alt="image" src="https://user-images.githubusercontent.com/9369625/231481896-36384e52-e2af-4291-81f8-195460b4d8e0.png">